docs/setup-instructions.md

```diff
@@ -0,0 +1,50 @@
+## Setup Instructions
+
+Please check our prerequisites [here](https://github.com/ARPA-Network/BLS-TSS-Network/blob/main/docs/eigenlayer-onboarding.md#prerequisites).
+
+### Step 1: Clone the repo
+
+```bash
+git clone https://github.com/ARPA-Network/ARPA-Network-AVS-Monitoring.git
+```
+
+### Step 2: Update configuration
+
+Update `config.yml` file in the root directory.
```

Comment:
Might be helpful to provide them with a sample config that is guaranteed to show them data for an existing node. Something like:

"The following test config can be used to follow along with this tutorial: "

```yaml
node_address: 0x1BD61FAAa74F7B2aAF414753B74d6070f821D16D
chain_id: 1
rpc_endpoint: https://mainnet.gateway.tenderly.co/4X987cMYL0drsf7LIZXlSr
```

If the user uses the above config.yaml, they should see a dashboard result similar to the image linked in my comment on line 50